src/ReasonReactErrorBoundary.re

```diff
@@ -11,32 +11,60 @@ type params('error) = {
   info,
 };
 
-[%%mel.raw
-  {|
-  var React = require("react");
-  var ErrorBoundary = (function (Component) {
-    function ErrorBoundary(props) {
-      Component.call(this);
-      this.state = {error: undefined};
-    };
-    ErrorBoundary.prototype = Object.create(Component.prototype);
-    ErrorBoundary.prototype.componentDidCatch = function(error, info) {
-      this.setState({error: {error: error, info: info}});
-    };
-    ErrorBoundary.prototype.render = function() {
-      return this.state.error != undefined
-        ? this.props.fallback(this.state.error)
-        : this.props.children
-    };
-    return ErrorBoundary;
-  })(React.Component);
-|}
-];
+[@mel.scope "Object"] external objCreate: 'a => Js.t({..}) = "create";
```

Comment:
If you want to add `melange.js`, can use Obj.empty

Reply:
`Js.Obj.empty` doesn't allow you to do `Object.create(React.Component.prototype)`.

Reply:
Ops, obviusly. Could add obj.create on `melange.js`